docs/src/appendix/forge-library/deserialize_json.md

```diff
@@ -0,0 +1,61 @@
+# `deserialize_json`
```

Comment:
we can document this, and link to cairo because Serde doesn't support it https://book.cairo-lang.org/appendix-03-derivable-traits.html?highlight=serde#store

Reply:
Fine by me, although wouldn't that be doable via a separate method? 
Something like, `load_multiple`
Up to discussion i guess, if you could make an issue it'd be great.

Reply:
I think we add an note to docs for now, and open an issue before this PR is merged so we can follow up.

Reply:
Ok, I read sth wrong, and we can support Array. I added it.